services/discourse_translator/microsoft.rb

```diff
@@ -14,38 +14,103 @@ class Microsoft < Base
     LENGTH_LIMIT = 5_000
 
     SUPPORTED_LANG = {
-      en: 'en',
-      en_US: 'en',
-      en_GB: 'en',
+      af: 'af',
+      am: 'am',
+      ar: 'ar',
+      as: 'as',
+      az: 'az',
+      ba: 'ba',
+      bg: 'bg',
+      bn: 'bn',
+      bo: 'bo',
+      bs: 'bs',
       bs_BA: 'bs-Latn',
+      ca: 'ca',
       cs: 'cs',
+      cy: 'cy',
       da: 'da',
       de: 'de',
-      ar: 'ar',
+      dv: 'dv',
+      el: 'el',
+      en: 'en',
+      en_US: 'en',
+      en_GB: 'en',
       es: 'es',
+      et: 'et',
+      eu: 'eu',
+      fa_IR: 'fa',
       fi: 'fi',
+      fj: 'fj',
       fr: 'fr',
+      fr_CA: 'fr-CA',
+      ga: 'ga',
+      gl: 'gl',
+      gu: 'gu',
       he: 'he',
+      hi: 'hi',
+      hr: 'hr',
+      ht: 'ht',
+      hu: 'hu',
+      hy: 'hy',
       id: 'id',
+      is: 'is',
       it: 'it',
+      iu: 'iu',
       ja: 'ja',
+      ka: 'ka',
+      kk: 'kk',
+      km: 'km',
+      kn: 'kn',
       ko: 'ko',
+      ku: 'ku',
+      ky: 'ky',
+      lo: 'lo',
+      lt: 'lt',
+      lv: 'lv',
+      mg: 'mg',
+      mi: 'mi',
+      mk: 'mk',
+      ml: 'ml',
+      mr: 'mr'
+      ms: 'ms',
+      mt: 'mt',
+      my: 'my',
+      nb_NO: 'nb',
+      ne: 'ne',
       nl: 'nl',
-      pt: 'pt',
+      or: 'or',
+      pa: 'pa',
+      pl_PL: 'pl',
+      ps: 'ps',
+      pt_BR: 'pt',
+      pt_PT: 'pt-PT',
```

Comment:
We don't have `pt_PT` in core, so this probably needs to be: 

```suggestion
      pt: 'pt-PT',
      pt_BR: 'pt',
```

Reply:
I applied your suggestion. However, know that before there was `pt: pt` in this list.